data/json/items/generic/filament.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "id": "thread",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "thread",
+    "description": "A small quantity of thread that could be used when sewing.",
+    "weight": 1,
+    "volume": "1ml",
```

Comment:
Pretty sure volume is for a full stack.
Volume for 1 thread was 5ml.
Don't specify stack_size and volume like that: stack size is a divisor on volume.